index.html

```diff
@@ -171,23 +239,38 @@ <h2 class="contact-title">Get started</h2>
             <p class="contact-description">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
                 tempor incididunt ut labore et
                 dolore magna.</p>
+            <p class="contact-description-desktop">
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
+                dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip
+                ex ea commodo consequat.
+            </p>
             <form action="https://formspree.io/f/mvodnypq" method="POST" class="actual-contact">
-                <div class="full-input">
-                    <label for="name">Your name</label>
-                    <input type="text" class="user-input" maxlength="30" name="name" id="name"
-                        placeholder="Enter your name here..." required>
-                </div>
-                <div class="full-input">
-                    <label for="email">Your email</label>
-                    <input type="email" class="user-input" name="email" id="email"
-                        placeholder="Enter your email here..." required>
+                <div class="input-container">
+                    <div class="full-input">
+                        <label class="label-text" for="name">Your name</label>
+                        <input type="text" class="user-input" maxlength="30" name="name" id="name"
+                            placeholder="Enter your name here..." required>
+                    </div>
+                    <div class="full-input">
+                        <label class="label-text" for="email">Your email</label>
+                        <input type="email" class="user-input" name="email" id="email"
+                            placeholder="Enter your email here..." required>
+                    </div>
                 </div>
+                <label class="textarea-label" for="user-message">Message</label>
                 <textarea name="user-message" id="user-message" cols="30" rows="10" maxlength="500"
                     placeholder="Enter your message here..."></textarea>
                 <button type="submit" class="submit-button">Start collaboration</button>
+                <div class="desktop-form-image">
+                    <img class="geo-image" src="./images/image-geometry_7.png"
+                        alt="This should be a colorful, geometric design">
+                </div>
             </form>
```

Comment:
- [ ] On the desktop screen, the form is not properly positioned, kindly inspect your media query to fix this issue.
![image](https://user-images.githubusercontent.com/33767951/139514664-0f21d6ea-76a1-4046-ba88-c41168dd2c68.png)